backend/papersfeed/tests/tests_paper.py

```diff
@@ -131,3 +131,35 @@ def test_put_paper_collection(self):
                               content_type='application/json')
 
         self.assertEqual(response.status_code, 200)
+
+    def test_paper_search(self):
+        """ PAPER SEARCH """
+        client = Client()
+
+        # Sign In
+        client.get('/api/session',
+                   data={
+                       constants.EMAIL: 'swpp@snu.ac.kr',
+                       constants.PASSWORD: 'iluvswpp1234'
+                   },
+                   content_type='application/json')
+
+        # Search with Keyword 'Computer'
+        response = client.get('/api/paper/search',
+                              data={
+                                  constants.TEXT: 'computer'
+                              },
+                              content_type='application/json')
+        self.assertEqual(response.status_code, 200)
+        # cs_500.json 파일 기준 415개 이다.
```

Comment:
LGTM, too.

By the way, I think we should modify tests related with Paper soon, because we may be going to remove 'cs_500.json' or 'cs_10.json' from our repository. Then, tests depending on the files won't work. So I think we should change tests so that they use test paper objects.

Reply:
@abcthing Ugh... the future came early. On incoming PR, the json fixture files may be deleted because json files including many Keyword and PaperKeyword records which are extracted from abstracts are too huge. From now, please create tests independent from fixture files.